BUFRCREX_TableB_en_13.csv

```diff
@@ -75,3 +75,10 @@ ClassNo,ClassName_en,FXY,ElementName_en,BUFR_Unit,BUFR_Scale,BUFR_ReferenceValue
 13,Hydrographic and hydrological elements,013162,Cloud liquid water,kg m-2,2,0,8,kg m-2,2,3,,,Operational
 13,Hydrographic and hydrological elements,013163,Snow water equivalent,kg m-2,0,0,16,kg m-2,0,5,,,Operational
 13,Hydrographic and hydrological elements,013164,Sea ice freeboard,m,3,-131072,18,m,3,6,,,Operational
+13,Hydrographic and hydrological elements,013165,CAPE (convective available potential energy),J/kg,0,0,14,J/kg,0,5,(see Note 11),issue-162a,Operational
+13,Hydrographic and hydrological elements,013166,Total totals index,K,1,0,10,C,0,5,(see Note 12),issue-162b,Operational
+13,Hydrographic and hydrological elements,013167,S index (STT),K,1,-1000,11,C,0,6,(see Note 13),issue-162c,Operational
+13,Hydrographic and hydrological elements,013168,Fog threat index,K,1,-100,8,C,0,5,(see Note 14),issue-162d,Operational
+13,Hydrographic and hydrological elements,013169,MDPI (microburst day potential index),K/K,1,-100,8,K/K,0,5,(see Note 15),issue-162e,Operational
+13,Hydrographic and hydrological elements,013170,TQ index,K,1,-1000,11,C,0,6,(see Note 16),issue-162f,Operational
+13,Hydrographic and hydrological elements,013171,CIN (convective inhibition),J/kg,0,-1000,10,J/kg,0,5,(see Note 17),issue-162g,Operational
```

Comment:
[@SibylleK](https://github.com/SibylleK) Units don't match in the issue summary and branch. Should they be "J/kg" or "Jkg-1"?

Reply:
dito